modifiers/string.go

```diff
@@ -99,6 +101,58 @@ func TitleCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param
 	return nil
 }
 
+var stripNumRegex = regexp.MustCompile("[^0-9]")
+
+// StripAlphaCase removes all non-numeric characters. Example: "the price is €30,38" -> "3038". Note: The struct field will remain a string. No type conversion takes place.
+// From https://github.com/leebenson/conform
+func StripAlphaCase(_ context.Context, _ *mold.Transformer, v reflect.Value, _ string) error {
+	s, ok := v.Interface().(string)
+	if !ok {
+		return nil
+	}
+	v.SetString(stripNumRegex.ReplaceAllLiteralString(s, ""))
+	return nil
+}
+
+var stripAlphaRegex = regexp.MustCompile("[0-9]")
+
+// StripNumCase removes all numbers. Example "39472349D34a34v69e8932747" -> "Dave". Note: The struct field will remain a string. No type conversion takes place.
+// From https://github.com/leebenson/conform
```

Comment:
I don't think these from statements are needed here if we can remove.

Reply:
Done.